### ISSUE TITLE
MSID: 89424 Version: 1 DOI: 2023.05.11.540467

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1021]
"msas": "Cell Biology"
"msid": "89424"
"preprintDoi": "10.1101/2023.05.11.540467"
"Reviewed Preprint posted": "2023-09-01"
"Sent for peer review": "2023-05-31"
"Posted to bioRxiv": "2023-07-13" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2023.05.11.540467v2/")

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=89424)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/89424/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [x] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.05.11.540467)
- [x] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/89424/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/89424)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/89424
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/89424
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/89424/v1/89424-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/89424)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by JGilbert-eLife** [comment]
Title is in all caps due to original version of preprint (https://www.biorxiv.org/content/10.1101/2023.05.11.540467v1). Note: eJP has different spelling of 'organization' (organisation), not sure if that's important.

Additionally, can we check which version of the preprint is required here? Posted to bioRxiv date given in ticket is for the second version, not the first version which has been pulled in (I think it should be the v1, 2023-05-14, based on sent for review date).

eLocations/single page numbers missing from reference list.

**Comment by fred-atherden** [comment]
Pretty certain we want the V2 loaded here.

I'll block, assign to myself and get the v2 loaded later today. It'll need another QC once the v2 is up. 

A note for the future - we should not be publishing articles with titles in all caps. In these cases we should either ask the authors to post a revised preprint (preferred option) or correct this ourselves via xslt.

**Comment by fred-atherden** [comment]
@JGilbert-eLife before we load the v2 preprint are you OK to reach out to the authors about this citation?

<img width="651" alt="Screenshot 2023-09-01 at 10 12 11" src="https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/d6fdb050-dfca-49db-81ff-1ac1f1359cb0">


I think ideally this is fixed prior to RP publication. Presumably they'd need to publish a new version of the preprint. 

(I only noticed because of the XML diff in GitHub when going to load the v2 version)

**Comment by JGilbert-eLife** [comment]
Oops, I should have spotted that! I've reached out to the corresponding author.

**Comment by fred-atherden** [comment]
No worries - many thanks!

**Comment by JGilbert-eLife** [comment]
The authors intend to sort this next week.

**Comment by fred-atherden** [comment]
full text for the v3 is up, but the MECA package is not yet available:

`curl 'https://api.biorxiv.org/meca_index/elife/all/2023.05.11.540467' | jq`


```json
{
  "status": [
    {
      "count": 2,
      "messages": "ok"
    }
  ],
  "results": [
    {
      "msid": "540467",
      "tdm_doi": "10.1101/2023.05.11.540467",
      "ms_version": "1",
      "filedate": "2023-05-18",
      "tdm_path": "s3://transfers-elife/biorxiv_Current_Content/May_2023/15_May_23_Batch_1612/299df481-6c3d-1014-aa31-c9bfb06bf035.meca",
      "transfer_type": "api"
    },
    {
      "msid": "540467",
      "tdm_doi": "10.1101/2023.05.11.540467",
      "ms_version": "2",
      "filedate": "2023-05-18",
      "tdm_path": "s3://transfers-elife/biorxiv_Current_Content/July_2023/15_Jul_23_Batch_1673/b8f39a02-6de0-1014-9cef-ff6b0774276d.meca",
      "transfer_type": "api"
    }
  ]
}
```

**Comment by fred-atherden** [comment]
v3 up at `s3://transfers-elife/biorxiv_Current_Content/September_2023/04_Sep_23_Batch_1724/e36d1cc6-6d4d-1014-8b99-beacfaa8f3d5.meca`

**Comment by fred-atherden** [comment]
@JGilbert-eLife this one should be ready to check again

**Comment by JGilbert-eLife** [comment]
Looks good, thanks!
